charts/git-server/chart/templates/cm-nginx.yaml

```
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx-conf
+data:
+  nginx.conf: |
+    user                    nginx;
+    worker_processes        4;
+    pid                     /run/nginx.pid;
+
+    events {
+        worker_connections    1024;
+    }
+
+    http {
+        sendfile              on;
+        tcp_nopush            on;
+        tcp_nodelay           on;
+        keepalive_timeout     65;
+        types_hash_max_size   2048;
+
+        include               /etc/nginx/mime.types;
+        default_type          application/octet-stream;
+
+        access_log            /dev/stdout;
+        error_log              /dev/stderr;
+
+        gzip                  on;
+        gzip_disable          "msie6";
+
+        server {
+            server_name       _;
+            listen 80         default_server;
+            listen [::]:80    default_server;
+
+            location /ping {
+                add_header Content-Type text/plain;
```

`default_type` is preferred as `add_header` has some [quirks](https://www.peterbe.com/plog/be-very-careful-with-your-add_header-in-nginx) (not really a issue in this case, but let's just go with `default_type`).
```suggestion
                default_type text/plain;
```